distribution/lib/Standard/Table/0.2.32-SNAPSHOT/src/Data/Sort_Method.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
+import Standard.Base.Data.Ordering.Sort_Order
+from Standard.Table.Data.Matching import Case_Insensitive
+
+## Specifies the sort methodology.
+
+   Arguments:
+   - order: Sort ordering - ascending by default.
+   - case_sensitive: Case sensitivity - case sensitive by default.
+   - natural_order: Whether to use natural ordering. Standard code point
+     ordering is used by default.
+type Sort_Method (order : Sort_Order = Sort_Order.Ascending) (case_sensitive : (True | Case_Insensitive) = True) (natural_order : Boolean = False)
+
+## UNSTABLE
+   A temporary workaround to allow the `Sort_Method` constructor to work with default arguments.
```

Comment:
Such comments should have better explanation of why this "temporary workaround" is needed and a link to an issue.